libglusterfs/src/glusterfs/inode.h

```diff
@@ -102,6 +102,7 @@ struct _inode {
     uuid_t gfid;
     gf_lock_t lock;
     gf_atomic_t nlookup;
+    gf_atomic_t kids;
```

Comment:
Children? Or not to confuse with AFR's children?

Reply:
Anything would do. I will keep 'kids' as the name for now, as its something new introduced. Children is used heavily in code already with FIRST_CHILD() macro, and in the graph code. Currently this is used as just for debug. Can be used in future enhancements if any.